core/src/main/java/com/linecorp/armeria/server/cors/CorsServiceBuilder.java

```diff
@@ -67,19 +68,20 @@ public static CorsServiceBuilder forAnyOrigin() {
      * Creates a new builder with the specified origin.
      */
     public static CorsServiceBuilder forOrigin(String origin) {
-        requireNonNull(origin, "origin");
-
-        if ("*".equals(origin)) {
-            return forAnyOrigin();
-        }
-        return new CorsServiceBuilder(origin);
+        return forOrigins(origin);
```

Comment:
`requireNonNull(origin, "origin");`?
because the message is different in `forOrigins`.